assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -212,7 +212,11 @@ function MediaPane(props) {
   };
 
   const filterResource = useCallback(
-    ({ mimeType, width, height }) => {
+    (resource) => {
+      if (!resource) {
```

Comment:
This seems like hiding a problem. Why would we have `null` resources? I'd rather we fix at the source.

Reply:
Fixed it so we don't have null resource.